src/main/java/org/broadinstitute/hellbender/tools/spark/BaseRecalibratorSparkSharded.java

```diff
@@ -43,16 +43,58 @@
 import java.util.List;
 import java.util.concurrent.TimeUnit;
 
+/**
+ * Experimental sharded spark implementation of the first pass of the base quality score recalibration.
+ * Generates recalibration table based on various covariates
+ * (such as read group, reported quality score, machine cycle, and nucleotide context).
+ *
+ * <p>
+ *
+ * <h3>Input</h3>
+ * <p>
+ * The input read data whose base quality scores need to be assessed.
+ * <p>
+ * A database of known polymorphic sites to skip over.
+ * </p>
+ * <p>
+ * <h3>Output</h3>
+ * <p>
+ * A GATK Report file with many tables:
+ * <ol>
+ * <li>The list of arguments</li>
+ * <li>The quantized qualities table</li>
+ * <li>The recalibration table by read group</li>
+ * <li>The recalibration table by quality score</li>
+ * <li>The recalibration table for all the optional covariates</li>
+ * </ol>
+ * <p>
+ * The GATK Report is intended to be easy to read by humans or computers. Check out the documentation of the GATKReport to learn how to manipulate this table.
+ * </p>
+ * <p>
+ * <h3>Examples</h3>
+ * <pre>
```

Comment:
Please update the program group to ReadDataProgramGroup

Examples --> Usage example
`./gatk` --> `gatk`

Please test your example commands to ensure they work. 

In the `All hands on deck` issue ticket, I tell Ted that if his commands work without explicitly specifying Spark options, then he should add the following blurb:

```
This tool can be run without explicitly specifying Spark options. That is to say, the given example command without Spark options will run locally. See <a href ="https://software.broadinstitute.org/gatk/documentation/article?id=10060">Tutorial#10060</a> for an example of how to set up and run a Spark tool on a cloud Spark cluster.
```

Reply:
Fixed